optuna/samplers/_tpe/multi_objective_sampler.py

```diff
@@ -32,8 +34,129 @@ def _default_weights_above(x: int) -> np.ndarray:
     return np.ones(x)
 
 
+class _ParzenEstimatorParameters(
+    NamedTuple(
+        "_ParzenEstimatorParameters",
+        [
+            ("consider_prior", bool),
+            ("prior_weight", Optional[float]),
+            ("consider_magic_clip", bool),
+            ("consider_endpoints", bool),
+            ("weights", Callable[[int], np.ndarray]),
+        ],
+    )
+):
+    pass
+
+
+class _ParzenEstimator(object):
```

Comment:
This class is copied from `optuna.samplers._tpe.parzen_estimator._ParzenEstimator`.